pytorch/notebooks/Serving PyTorch Models with CMLE  Custom Prediction Code.ipynb

```diff
@@ -0,0 +1 @@
CKET}/{GCS_MODEL_DIR} \\\n","            --runtime-version={RUNTIME_VERSION} \\\n","            --framework='SCIKIT_LEARN' \\\n","            --python-version=2.7 \\\n","            --package-uris=gs://{BUCKET}/{GCS_PACKAGE_URI}\\\n","            --model-class={MODEL_CLASS}"],"execution_count":0,"outputs":[]},{"metadata":{"id":"naG7FtQokRzM","colab_type":"code","colab":{"base_uri":"https://localhost:8080/","height":51},"outputId":"2df0240e-1445-47e8-9a41-627dfacab273","executionInfo":{"status":"ok","timestamp":1538246126033,"user_tz":-60,"elapsed":3411,"user":{"displayName":"Khalid Salama","photoUrl":"","userId":"16897751573918541504"}}},"cell_type":"code","source":["!gcloud ml-engine versions list --model {MODEL_NAME}"],"execution_count":24,"outputs":[{"output_type":"stream","text":["NAME  DEPLOYMENT_URI                                            STATE\n","v1    gs://ksalama-gcs-cloudml/models/pytorch/iris_classifier/  READY\n"],"name":"stdout"}]},{"metadata":{"id":"TQz-JaqpkRzP","colab_type":"text"},"cell_type":"markdown","source":["# D. CMLE Online Prediction"]},{"metadata":{"id":"Wdx5nTxjkRzR","colab_type":"code","colab":{}},"cell_type":"code","source":["from googleapiclient import discovery\n","from oauth2client.client import GoogleCredentials\n","\n","credentials = GoogleCredentials.get_application_default()\n","api = discovery.build('ml', 'v1', credentials=credentials,\n","                      discoveryServiceUrl='https://storage.googleapis.com/cloud-ml/discovery/ml_v1_discovery.json')\n","\n","\n","def estimate(project, model_name, version, instances):\n","    \n","    request_data = {'instances': instances}\n","\n","    model_url = 'projects/{}/models/{}/versions/{}'.format(project, model_name, version)\n","    response = api.projects().predict(body=request_data, name=model_url).execute()\n","\n","    #print response\n","    \n","    predictions = response[\"predictions\"]\n","    return predictions"],"execution_count":0,"outputs":[]},{"metadata":{"id":"ETKlF4yHkRzT","colab_type":"code","colab":{"base_uri":"https://localhost:8080/","height":34},"outputId":"f1089ffd-c6c0-4db9-cf64-02bf414a5701","executionInfo":{"status":"ok","timestamp":1538246147364,"user_tz":-60,"elapsed":869,"user":{"displayName":"Khalid Salama","photoUrl":"","userId":"16897751573918541504"}}},"cell_type":"code","source":["instances = [\n","    [6.8, 2.8, 4.8, 1.4],\n","    [6. , 3.4, 4.5, 1.6]\n","]\n","\n","predictions = estimate(instances=instances\n","                     ,project=PROJECT\n","                     ,model_name=MODEL_NAME\n","                     ,version=MODEL_VERSION)\n","\n","print(predictions)"],"execution_count":27,"outputs":[{"output_type":"stream","text":["[u'versicolor', u'versicolor']\n"],"name":"stdout"}]},{"metadata":{"id":"nk8oZ2zPkRzW","colab_type":"text"},"cell_type":"markdown","source":["# Questions? Feedback?\n","Feel free to send us an email (cloudml-feedback@google.com) if you run into any issues or have any questions/feedback!"]}]}
```

Comment:
1. For new notebooks, we're pushing
2. You're opening paragraph mentions scikit-learn, but it isn't used anywhere throughout the tutorial, is that mention relevant or what feature is new in CMLE that is related to why PyTorch now works? Custom Prediction Routine? 
3. In setup, there should be steps to creating a project and setting up all the required things needed to run this on GCP. Similar to what is in the [ml engine training notebook](https://github.com/GoogleCloudPlatform/cloudml-samples/blob/master/sklearn/notebooks/ml_engine_training.ipynb) unless you only plan to have them use this in colab, then there should be instructions for how to set that up. 
4. Under Setup in code block 3. There shouldn't be any identifiable information about `PROJECT`, `BUCKET`, rather use `YOUR_PROJECT_ID` and `BUCKET_ID`. 
5. Under the numbered sections, mind adding a few sentences to provide the user with an explanation of what is happening, such as in "1. Load Data to Pandas Dataframes" what are the features / labels, why numerical is needed. Or in "2 Set model parameters" what each of the parameters mean for this model and possibly why the values were chosen. 
6. In step 6 you test the model, but don't verify if the test is accurate. 
7. Do we have any docs about the Custom Prediction Package that can be linked to? or a few sentence overview of what this is and why its important.